client/src/components/NavLink/NavLink.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import NavLink from './NavLink';
+
+const placeholderTextProp = "COMMUNITY";
+const placeholderUrl = "community page route";
+const placeholderDropdownProp = [
+  {
+    text: 'Explore People',
+    url: 'community people route'
+  },
+  {
+    text: 'Join Groups',
+    url: 'community groups route'
+  }
+]
```

Comment:
This test data is great - can you change the `url` properties to be URLs though?